self-core-impl/src/main/java/com/selfxdsd/core/AccessToken.java

```diff
@@ -83,4 +83,35 @@ public String value() {
             return "Bearer " + this.value;
         }
     }
+
+    /**
+     * A Bitbucket <a href="https://developer.atlassian.com/bitbucket/api/2/reference/meta/authentication#make-requests">access token</a>.
+     *
+     */
+    class Bitbucket implements AccessToken {
+
+        /**
+         * Header value.
+         */
+        private final String value;
+
+        /**
+         * Ctor.
+         *
+         * @param value Header value.
+         */
+        public Bitbucket(final String value) {
```

Comment:
@amihaiemil Unlike gh/gl, bitbucket tokens expire in 1h https://developer.atlassian.com/bitbucket/api/2/reference/meta/authentication#refresh-tokens.  We may need to pass the 'value' through a `Supplier` that will ensure the token is always up to date.

Reply:
@criske good point, I'll keep it noted.